docs/dev_mode.md

```diff
@@ -8,33 +8,19 @@ Arbitrary agent services built with the {{open_autonomy}} framework can be run w
 
 ### Environment setup
 
-Export the following environment variables (where you replace the paths with valid ones):
-```bash
-export OPEN_AEA_REPO_PATH="/path/to/open-aea-repo"
-export DEPLOYMENT_TYPE="docker-compose"
-export DEPLOYMENT_SPEC="/path/to/deployment-spec"
-export VERSION=dev
-```
-
-The default deployment specifications are stored in `deployments/deployment_specifications`.
-
-The `open-aea` repository can be cloned from here: https://github.com/valory-xyz/open-aea
+Fetch the required service or if you already have a service defined navigate to the service directoy and run
 
-*Tip: You can also store above variables in your .bashrc file*
-
-
-### Prepare images & run
-
-Execute the following:
 ```bash
-make build-deploy
+# build dev deployment using
+$ autonomy deploy build --dev --packages-dir PATH_TO_PACKAGES_DIR --open-aea-dir PATH_TO_LOCAL_OPEN_AEA_REPO --open-autonomy-dir PATH_TO_LOCAL_OPEN_AUTONOMY_DIR
+# build the required images using
+$ autonomy build-images --dependencies
+$ autonomy build-images --dev
 ```
-This builds the latest docker images and launches docker compose. Wait for environment setup to be completed inside your docker containers, it may take some time. Also don't make any changes on either open-aea or open-autonomy repository while environment is being set up it may cause some unexpected errors.
 
-Then run:
-```bash
-make run-deploy
-```
+This will create a deployment with hot reload enabled for agents. You can run it using the same methods as normal deployments. Use `autonomy deploy run` or `docker-compose up --force-recreate` to start the deployment and enjoy buiding the services.
```

Comment:
Here you mention hot reload, below hot reload has been changed to dev mode. Shouldnt we update here as well?

Reply:
Hot reload is just a feature which can be used in the dev mode. The dev mode is a setup which provides various features (Right now it's hot reload, storing data for replay and benchmarking).